kivy/core/camera/camera_avfoundation_implem.m

```diff
@@ -426,13 +441,56 @@ The user can override these settings by calling the attemptCapturePreset() funct
     return 0;
 }
 
-bool Camera::setVideoOrientation(int orientation) {
+void Camera::setVideoOrientation(int orientation) {
     AVCaptureConnection *conn = [mCaptureDecompressedVideoOutput connectionWithMediaType:AVMediaTypeVideo];
     if([conn isVideoOrientationSupported]){
         [conn setVideoOrientation:(AVCaptureVideoOrientation) orientation];
     }
 }
 
+#if TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR
+void Camera::changeCameraInput(int _cameraNum) {
+    NSError* error;
+    NSArray *devices;
+    AVCaptureDevice *device;
+
+    devices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
+
+    if (_cameraNum >= 0) {
+        int camNum = _cameraNum % [devices count];
+        if (camNum != _cameraNum) {
+            NSLog(@"Warning: Max Camera Num is %lu; Using camera %d\n",
+                  (unsigned long)([devices count] - 1), camNum);
+        }
+        device = [devices objectAtIndex:camNum];
+        cameraNum = _cameraNum;
+    } else {
+        device = [AVCaptureDevice defaultDeviceWithMediaType:AVMediaTypeVideo]  ;
+        cameraNum = 0;
+    }
```

Comment:
The new  [discoverySessionWithDeviceTypes:mediaType:position:](https://developer.apple.com/documentation/avfoundation/avcapturedevicediscoverysession/2361539-discoverysessionwithdevicetypes) allows a fairly more granular selection of the camera. 

The `devices` array may be not consistent of returning the "correct" order of the cameras, so could be nice to create an enumerator on Kivy side, and pass the configuration requested from the user to  `discoverySessionWithDeviceTypes:mediaType:position:`